packages/regl-worldview/src/Worldview.js

```diff
@@ -37,13 +37,15 @@ type ControlledType = {|
   ...SharedProps,
   ...Dimensions,
   cameraState: CameraState,
+  keyMap?: KeyMapping,
```

Comment:
what do you think about changing `KeyMapping` to `KeyMap` just to be consistent?